app/services/foreman_ansible/roles_importer.rb

```diff
@@ -1,24 +1,55 @@
 module ForemanAnsible
-  # Service to import roles from the filesystem.
-  # Should be extracted to a gem that both the proxy and Foreman can use.
+  # imports roles from smart proxy
```

Comment:
Read the previous comment, there are users already using this - the importer should handle both the local fs & the smart proxy. Maybe it's best to split that functionality into two classes